tests/karma/unit/controllers/contacts-content.js

```diff
@@ -221,5 +236,21 @@ describe('ContactsContentCtrl', function() {
       });
     });
 
+    it('includes subjectIds in reports search so JSON reports are found', function(done) {
+      childContactPerson.patient_id = 'cd';
+      childContactPerson.place_id = 'ef';
+      stubSearch(null, [ { _id: 'ab' } ]);
+      runPersonTest(done, doc, null, function(selected) {
+        chai.expect(selected.reports.length).to.equal(1);
+        chai.expect(selected.reports[0]._id).to.equal('ab');
+        chai.expect(search.callCount).to.equal(1);
+        chai.expect(search.args[0][0]).to.equal('reports');
+        chai.expect(search.args[0][1].subjectIds.length).to.equal(3);
```

Comment:
I prefer the expect syntax, but admit it's not for everyone. Maybe we need to talk about that and come up with one syntax for all for consistency. It's painful to mix and match too much...